tools/testing/gatekeeper/jobs.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Keeps checking the current PR until all required jobs pass
+Env variables:
+* REQUIRED_JOBS: comma separated list of required jobs
+* REQUIRED_REGEXPS: comma separated list of regexps for required jobs
+* COMMIT_HASH: Full commit hash we want to be watching
+* GITHUB_REPOSITORY: Github repository (user/repo)
+Sample execution (GH token can be excluded):
+GITHUB_TOKEN="..." REQUIRED_JOBS="skipper / skipper"
```

Comment:
Apologies in advance if I'm just being boneheaded but line 10 above says that `REQUIRED_JOBS` should be a comma-separated list which `"skipper / skipper"` doesn't seem to be.

Reply:
It is, the full name of skipper job is `skipper / skipper` as it includes the `workflow_name`.